### PR TITLE
revamped footer

### PR DESCRIPTION
**Pull request by harry-hov**
# Updated Footer
Screenshot (88)

List of Changes:
- "Contact Us" section is now moved to footer.
- Links to all the social platforms
- Developer and Maintainer section
- Click on Image reveals their name and GitHub Username with embedded link
Screenshot (91)

### CODE REVIEW

**Comment by mooocer** [comment]
You forgot to open an issue despite reminding twice. We don't use issues so far but we have to start using issues for development.
Issues help keep track of TODOs before someone opens a PR. They help know if people are working on it or not.

**Comment by harry-hov** [comment]
fixes #23

**Comment by mooocer** [comment]
Really nice work @harry-hov :clap:

Sorry for the delay in review. Here's a couple of things that need to be changed:
1. There's a lot of empty space below the faces
2. Faces need to be smaller, such that they don't draw too much attention but still be there for attribution.
3. Names should be only first names
4. Copyright text should be centered and at the bottom
5. Remove social media links that don't have much traction so far (Twitter and LinkedIn)
6. Names shouldn't appear sidewise but at the bottom, maybe by raising the icon a little. This is because when the second-last image is clicked it pushes the last one down and it doesn't look good.

Extra:
1. Instead of activating by click, can we use hover to show the text?
2. Can we keep the contact section as it is for now? Is there anything else we can put in the footer instead?

File: index.html [hunk 1]
```diff
@@ -38,6 +38,9 @@
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"/>
     <!-- Font Awesome -->
     <script src="https://use.fontawesome.com/901f1cce32.js"></script>
+    <!--
+    <script defer src="js/fontawesome-all.min.js"></script>
```
Comment (mooocer) [inline index.html]: Why commented code?

File: index.html [hunk 2]
```diff
@@ -223,40 +226,85 @@ <h1>hackSocIndore</h1>
...
+                            <div class="col-sm-2">
+                                <a href="#hariom" data-toggle="collapse">
+                                    <img src="img/devlopers/hariom.jpg" class="rounded-circle person" alt="Hariom" width="90" height="90">
+                                </a>
+                            </div>
+                                <div id="hariom" class="collapse" style="margin-top: 2em">
+                                    <p>Name: Hariom Verma</p>
+                                    <p>Github: <a href="https://github.com/harry-hov">@harry-hov</a></p>
+                                </div>
+                        </div>
```
Comment (mooocer) [inline index.html:262]: Are you sure the indentation is uniform?

File: index.html (hunk 1 above)
Comment (harry-hov) [reply index.html]: I'll remove that

File: index.html (hunk 2 above)
Comment (harry-hov) [reply index.html]: @mishuvs I'll resolve the comments, once my midsem gets over. And will implement the same suggested by you

**Comment by harry-hov** [comment]
@mishuvs  I'll resolve the comments, once my midsem gets over. And will implement the same suggested by you

**Comment by harry-hov** [comment]
Ready for review @mishuvs

**Review by shashank-sharma — APPROVED** [review]
Right now the footer is looking clean and better than last one
Further enhancement can be done in later Pull Request

**Review by mooocer — APPROVED** [review]
Nice! :tada:
Some improvement needed, but let's handle it in some other PR.
There are 3 people whose names didn't get included in this PR. I'll open an issue for that.

**Pull request merged**